### DISCUSSION TITLE
Horizontal screen option

### DISCUSSION THREAD

**Discussion by Dylotic** [Jyers/Marlin#525] [Ideas]
I would like to mount my lcd horizontal and have all the benefits of this firmware. Is it possible to have a ver/hor toggle option.

**Comment by aryajuanda** [comment]
Its posible to have a landscape orientation, but you must recompile and adjust the location of each icon to get a good view.. 

I have saw some one made a landscape firmware for ender 3v2 before.. 

https://otioss.com/posts/ender3-v2-front-panel

**Reply by Dylotic** [reply]
Could I mix this firmware with his DWIN and have all the features?

**Reply by Jyers** [reply]
I had discussed with him porting over his layout to this firmware, but I haven't heard back in a while. I'll check in to see if there has been any progress.

**Reply by Dylotic** [reply]
If I would like to adjust the firmware myself for landscape.
Where are the positions in the dwin code?

**Reply by Jyers** [reply]
@Dylotic It's a lot of work to adjust the positions as everything needs to be moved/resized and new elements need to be made. Hopefully this will get integrated soon.

**Reply by JetsonPaul** [reply]
Hi guys, 
I did the otioss upgrade to my printer because the view angle from the left side was so bad on my screen. Now it’s really nice but I’m missing some features of your firmware. Are there any news or progress on landscape mode? 
best regards Paul

**Reply by agalakhov** [reply]
My refactoring contains possibility for easy landscape support. Stay tuned.
